api-tests/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "api-tests",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Saleor API tests",
+  "main": "N/A",
+  "scripts": {
+    "test": "vitest",
+    "lint": "eslint 'src/**/*.@(tsx|ts|jsx|js)' --fix",
+    "check-types": "tsc --noEmit",
+    "generate": "graphql-codegen"
+  },
+  "author": "Saleor Commerce",
+  "license": "BSD",
+  "dependencies": {
+    "date-fns": "2.30.0",
+    "eslint": "8.41.0",
+    "eslint-config-prettier": "8.8.0",
+    "eslint-plugin-import": "2.27.5",
+    "eslint-plugin-no-only-tests": "3.1.0",
+    "eslint-plugin-vitest": "0.2.3"
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "5.59.6",
+    "@vitest/ui": "0.31.1",
+    "eslint-plugin-chai-friendly": "0.7.2",
+    "eslint-plugin-formatjs": "4.10.1",
+    "eslint-plugin-no-skip-tests": "1.1.0",
+    "eslint-plugin-prettier": "4.2.1",
+    "eslint-plugin-simple-import-sort": "10.0.0",
+    "prettier": "2.8.8",
+    "typescript": "5.0.4",
+    "vite": "4.3.8",
```

Comment:
As @krzysztofzuraw originally suggested, I would also change pinned package versions to have automatic patches and minor updates instead. It helps to stay up-to-date with bug fixes and security releases.

Reply:
https://github.com/saleor/saleor/pull/12963/commits/a3e557cbfa51372de84978926f7049e267d38b19